config/kubernetes/sig-release/teams.yaml

```
@@ -281,27 +281,52 @@ teams:
         maintainers:
         - palnabarun # subproject owner (1.21 RT Lead)
         members:
+        - AnaMMedina21 # 1.26 Release Notes Shadow
+        - Atharva-Shinde # 1.26 Enhancements Shadow
+        - cailynse # 1.26 Bug Triage Shadow
+        - cathchu # 1.26 Docs Shadow
         - cici37 # subproject owner (1.25 RT Lead)
         - cpanato # subproject owner / Technical Lead
+        - bharitwal # 1.26 CI Signal Shadow
+        - bradmccoydev # 1.26 Comms Shadow
```

pull-org-test-all test is failing due to these entries are not sorted
please move the members starting with the letter b before the members starting with the letter c
```suggestion
        - bharitwal # 1.26 CI Signal Shadow
        - bradmccoydev # 1.26 Comms Shadow
```